src/hazelcore/semantics/ErrStatus.re

```diff
@@ -6,6 +6,16 @@ module HoleReason = {
     | WrongLength;
 };
 
+module RunTime = {
```

Comment:
don't make it a submodule of ErrStatus, since ErrStatus is about an entirely different kind of error. just create a InvalidOperationError module in the dynamics directory